### ISSUE TITLE
Family Filter: Block proxy sites

### ISSUE THREAD

**Issue by Esokrates** [AdguardTeam/AdGuardHome#267]
Please consider blacklisting proxysites that help to circumvent the family restrictions. 
Those proxies are easy to research even for kids, usually the suggestion to use a proxy service is the first to come up when googling how to circumvent filters.

**Comment by ibksturm** [comment]
hi @Esokrates 
i'm myself dad of two wunderful little childrens... i blocked porn, ad malware etc. pages first on my own dns infrastructur and second on two dns dns resolvers (adguard and cleanbrowsing -> all over dnscrypt)

that's not exactly my understanding from freedom of the internet... but freedom of housekeeping (wife.......)

so i've got a other point of view on proxys and tor relays and this kind of thing... if my children (and i hope this day will come) figure out how to handle proxys etc. f**** up censorship, i'm sure and proud as daddy to teach them every right...

as i say on my own point of view, other idears are welcome :)

**Comment by Esokrates** [comment]
@ibksturm I agree with you, I have thought about that too, especially I remember myself as a student in school I got around the firewall there too, not to do something inappropriate, but because of freedom.

But I see it this way: I do not enforce the DNS Server, I set it in my router through DHCP. The clean way around the restriction would be to set a different DNS Server and not use shady Proxy servers.

I see it this way: They are very young and should not see certain things (talking here about 10 years and younger) and I think it may be bad to open the door to easy. Using an online proxy service is way too easy to accomplish. If they are at the level to be able to research what else to try other than that I think they are ready anyway to access whatever they want. 

Anyway I consider this issue to be a lower priority one, but technically its way to easy to render the filtering completely useless without any knowledge at all.
